.ci/Jenkinsfile

```diff
@@ -238,6 +238,44 @@ pipeline {
         }
       }
     }
+    stage('Sync K8s') {
+      when {
+        // Only on Prs
+        // Enable if k8s related changes.
+        anyOf {
+          changeRequest()                          // If is PR # This needs to be tested after merging in main with not { changeRequest() } and branch 'master' 
```

Comment:
```suggestion
          changeRequest()                          // If is PR # This needs to be tested after merging in main with not { changeRequest() } and branch 'main'

Reply:
Updated to run only in main: on commit 8701c1064629711160d34ac3c4e9fea3840c83c6